feature-libs/customer-ticketing/assets/translations/en/customer-ticketing.ts

```diff
@@ -21,5 +21,8 @@ export const customerTicketing = {
     category: 'Category',
     associateTo: 'Associate To',
     attachments: 'Attachments',
+    RequestReopened: 'Request Reopened.',
+    RequestClosed: 'Request closed, you may reopen if needed.',
```

Comment:
`requestClosed`, `requestReopened`

Reply:
> Let's not rename customer-ticketing-service to details if it's going to stay in the components folder! if we are moving it inside details then I agree with the renaming of the file

Once all the PRs are merged, we are going to restructure this components into details and list view. and this service would be inside details.